compiler/noirc_frontend/src/hir/comptime/interpreter/builtin.rs

```diff
@@ -826,6 +882,13 @@ fn modulus_num_bits(
     Ok(Value::U64(bits))
 }
 
+// fn quoted_eq(_first: Quoted, _second: Quoted) -> bool
+fn quoted_eq(arguments: Vec<(Value, Location)>, location: Location) -> IResult<Value> {
+    let (self_value, other_value) = check_two_arguments(arguments, location)?;
+
+    Ok(Value::Bool(self_value == other_value))
```

Comment:
Not necessary, but it could be nice to have a sanity-check that `self_value, other_value` are `Quoted`'s:
```suggestion
    assert_eq!(self_value.get_type() == Cow::Owned(Type::Quoted(QuotedType::Quoted)));
    assert_eq!(other_value.get_type() == Cow::Owned(Type::Quoted(QuotedType::Quoted)));
    Ok(Value::Bool(self_value == other_value))
```

Reply:
We can type check with `get_quoted` but we can't assert here - there may still be type errors in code ran by the interpreter

Reply:
Done! I also made a similar thing in `type_eq`